docs/sources/contribute/release-notes/index.md

```diff
@@ -34,185 +34,104 @@ The Grafana Cloud Traces updates are published using the What's new.
-           <p>If you need more information on adding an image, refer to <a href="https://grafana.com/docs/writers-toolkit/write/image-guidelines/">Image, diagram, and screenshot guidelines</a>.</p>
-           <p>If you need to mention a feature flag, use this format: To try out Trace to profiles, enable the <code>traceToProfile</code> <a href="https://grafana.com/docs/grafana/latest/setup-grafana/configure-grafana/feature-toggles/">feature toggle</a>.</p>
-         </td>
-       </tr>
-       <tr>
-         <th scope="row">DOCUMENTATION URL (OPTIONAL)</th>
-         <td>URL to the public documentation for this feature.</td>
-         <td></td>
-       </tr>
-       <tr>
-         <th scope="row">ENABLEMENT VIDEO (OPTIONAL)</th>
-         <td>Link to the video used for enablement.</td>
-         <td>
-           <p>Enablement videos are perhaps the fastest, most engaging tool for employees and users to learn about your feature. Use them for maximum engagement.</p>
-           <p>Follow these instructions to create and upload a video: <a href="https://docs.google.com/document/d/1nCiG62FxJ9J_qLTnlzNopSsT-VEAlxCUCsrAEpWL68U/edit#heading=h.fierz9i4q8ft">Enablement video instructions</a>.</p>
-           <p>When you upload an enablement video, the Content team receives a notification, edits it for the public, and uploads it to YouTube to coincide with your feature's release. They need a few weeks' lead time for this.</p>
-         </td>
-       </tr>
-       <tr>
-         <th scope="row">INTERNAL INFORMATION (OPTIONAL)</th>
-         <td>Information for Grafana Labs employees only.</td>
-         <td>
-           <p>For example, ProductDNA, Slack channel, FAQ, training documentation, or videos.</p>
-           <p>Used for training and internal announcements.</p>
-           <p>This is only visible on the <a href="https://admin.grafana.com/whats-new/">internal What's new page</a>, <em>not</em> the <a href="https://grafana.com/docs/grafana-cloud/whats-new/">public What's New page</a>.</p>
-         </td>
-       </tr>
-     </tbody>
-   </table>
+When you're ready to add a What's new entry, complete the following steps:
 
+1. Go to the What's new collection of the [website CMS](https://admin.grafana.com/content-admin/#/collections/whats-new) and click **New Feature** at the top of the page.
+1. Fill out [the CMS fields](#cms-fields).
 1. Click **Save**.
    The entry is now in **Draft** status and the CMS opens a pull request in the `grafana/website` repository.
 
-   If your entry is future-dated, it won't show up in the website-generated preview or in your local build, but you can see it in the preview of the internal feed.
-   To do so, remove `/docs/grafana-cloud` from the preview URL and add the heading for your entry at the end of the URL.
-   It should look something like this: `https://deploy-preview-18347-zb444pucvq-uw.a.run.app/whats-new/#create-subtables-in-table-visualizations`.
+1. (Optional) Preview your entry on GitHub.
```

Comment:
@chri2547 - Hopefully this is clearer. Further explanation on line 72.

Reply:
👍 , clearer - thanks!